scrapy/templates/project/module/settings.py.tmpl

```diff
@@ -2,16 +2,83 @@
 
 # Scrapy settings for $project_name project
 #
-# For simplicity, this file contains only the most important settings by
-# default. All the other settings are documented here:
+# For simplicity, this file contains only settings considered important or
+# commonly used. You can find more settings consulting the documentation:
 #
 #     http://doc.scrapy.org/en/latest/topics/settings.html
-#
+#     http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
+#     http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
 
 BOT_NAME = '$project_name'
 
 SPIDER_MODULES = ['$project_name.spiders']
 NEWSPIDER_MODULE = '$project_name.spiders'
 
+
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
 #USER_AGENT = '$project_name (+http://www.yourdomain.com)'
+
+# Configure maximum concurrent requests performed by Scrapy (default: 16)
+# CONCURRENT_REQUESTS=32
+
+# Configure a delay for requests for the same website (default: 0)
```

Comment:
I think "website" is fine - it could mean either "ip" or "domain" depending on settings. Slots are not documented, so investigation won't help :)